pkg/router/template/router.go

```diff
@@ -428,9 +464,19 @@ func (r *templateRouter) writeDefaultCert() error {
 // watchMutualTLSCert watches the directory containing the certificates for
 // mutual TLS and reloads the router if the directory contents change.
 func (r *templateRouter) watchMutualTLSCert() error {
+	var caUpdateChannel chan struct{}
 	caPath := os.Getenv("ROUTER_MUTUAL_TLS_AUTH_CA")
+	// TODO: Determine the best place to put CRLs
+	crlPath := "/crls/crls.pem"
+	os.Setenv("ROUTER_MUTUAL_TLS_AUTH_CRL", crlPath)
 	if len(caPath) != 0 {
+		caUpdateChannel = make(chan struct{})
```

Comment:
You might as well declare `caUpdateChannel` inside the `if` block that uses it.
```suggestion
	caPath := os.Getenv("ROUTER_MUTUAL_TLS_AUTH_CA")
	// TODO: Determine the best place to put CRLs
	crlPath := "/crls/crls.pem"
	os.Setenv("ROUTER_MUTUAL_TLS_AUTH_CRL", crlPath)
	if len(caPath) != 0 {
		caUpdateChannel := make(chan struct{})
```